api/src/github.rs

```diff
@@ -108,25 +108,6 @@ impl GitHubUserClient {
     let repo: Repository = res.json().await?;
     Ok(Some(repo))
   }
-
-  #[instrument(name = "GitHubUserClient::get_repo_by_id", skip(self), err)]
-  pub async fn get_repo_by_id(
```

Comment:
Is this dead code that isn't used anywhere?

Reply:
no :)